src/models/tools/geometry/geometry-content.ts

```diff
@@ -8,18 +8,20 @@ import * as uuid from "uuid/v4";
 export const kGeometryToolID = "Geometry";
 
 export const kGeometryDefaultHeight = 200;
+// matches curriculum images
+export const kGeometryDefaultPixelsPerUnit = 35.5;
+export const kGeometryDefaultAxisMin = -1;
 
 export function defaultGeometryContent(overrides?: JXGProperties) {
-  const axisMin = -0.5;
-  const xAxisMax = 20;
-  const yAxisMax = 5;
+  const xAxisMax = 30;
+  const yAxisMax = kGeometryDefaultHeight / kGeometryDefaultPixelsPerUnit - kGeometryDefaultAxisMin;
```

Comment:
As we guessed, the units do scale down once the grid approaches 5x5! I think that's a fine and very edge-case behavior

Reply:
It might not be so uncommon once we can create multiple tiles on a row, but for now it should be ok.